### ISSUE TITLE
Couldn't resolve "cee2ee7bccf83a7776c85588759aa0b8" as either a commit, a tag, or a head

### ISSUE THREAD

**Issue by c-ranger** [MaxLeiter/sortablejs-vue3#99]
Please help!

I can no longer install sortablejs-vue3

```
C:\Users\User\Projects\projectName>yarn add sortablejs-vue3 sortablejs
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: sortablejs@github:roy-mdr/Sortable#cee2ee7bccf83a7776c85588759aa0b8: Couldn't resolve "cee2ee7bccf83a7776c85588759aa0b8" as either a commit, a tag, or a head - if a commit, use the 40-characters commit hash
    at a (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:593:6599)
    at hO (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:593:6928)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Mb.getCandidates (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:593:11856)
    at async kf.getCandidates (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:391:1224)
    at async kf.getCandidates (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:391:1224)
    at async C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:439:8025
    at async df (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:390:11064)
    at async ge (C:\Users\User\Projects\projectName\.yarn\releases\yarn-3.6.4.cjs:439:8007)
    at async Promise.allSettled (index 1)
➤ YN0000: └ Completed in 1s 587ms
➤ YN0000: Failed with errors in 1s 590ms
```

**Comment by c-ranger** [comment]
I found a workaround:

Add this to package.json (without the caret '^' before the version number):

`"sortablejs-vue3": "1.2.9",`

**Comment by therealstein** [comment]
some ai help....

This warning message is related to peer dependencies in npm. Peer dependencies are a specific kind of dependency that's expected to be installed into the parent project by the user, not as a child of the package that defines the peer dependency.

In your case, the sortablejs-vue3 package has a peer dependency on sortablejs at a specific commit (github:roy-mdr/Sortable#cee2ee7bccf83a7776c85588759aa0b8), but the installed version of sortablejs in your project is 1.15.0, which doesn't satisfy the peer dependency requirement.

**Comment by Tnze** [comment]
Same

**Comment by MaxLeiter** [comment]
This should be resolved in v1.2.11.
